spark/pom.xml

```diff
@@ -42,6 +42,8 @@
     <hadoop.version>2.3.0</hadoop.version>
     <yarn.version>${hadoop.version}</yarn.version>
     <jets3t.version>0.9.3</jets3t.version>
+
+    <spark.download.url>http://www.apache.org/dist/spark/spark-${spark.version}/spark-${spark.version}.tgz</spark.download.url>
```

Comment:
@bzz I've found the line below in my failed test.

```
[INFO] --- download-maven-plugin:1.2.1:wget (download-pyspark-files) @ zeppelin-spark ---
[INFO] Got from cache: /home/travis/.m2/repository/.cache/maven-download-plugin/spark-1.3.1.tgz
[INFO] Expanding: /home/travis/build/apache/incubator-zeppelin/spark/target/spark-dist/spark-1.3.1.tgz into /home/travis/build/apache/incubator-zeppelin/spark/target/spark-dist
```
You have a question about this situation, don't you?